### DISCUSSION TITLE
[9.x] Translate additional error message summary from the validation errors

### DISCUSSION THREAD

**Discussion by vmartins** [laravel/framework#41230] [Ideas]
Cannot translate additional validation error message.

[ValidationException.php#L99](https://github.com/laravel/framework/blob/v9.2.0/src/Illuminate/Validation/ValidationException.php#L86)

Suggestion:
```php
...

if ($additional = count($messages)) {
    $pluralized = 1 === $additional ? __('error') : __('errors');

    $message .= __(" (and :additional more :pluralized)", compact('additional', 'pluralized'));
}

...
```

**Comment by skarjoss** [comment]
> Cannot translate additional validation error message.
> 
> [ValidationException.php#L99](https://github.com/laravel/framework/blob/v9.2.0/src/Illuminate/Validation/ValidationException.php#L86)
> 
> Suggestion:
> 
> ```
> ...
> 
> if ($additional = count($messages)) {
>     $pluralized = 1 === $additional ? __('error') : __('errors');
> 
>     $message .= __(" (and :additional more :pluralized)", compact('additional', 'pluralized'));
> }
> 
> ...
> ```

PR #43584

**Comment by Pablo-Camara** [comment]
I am having the same issue, how was this solved? did you manage to solve it elegantly ?
<img width="294" alt="image" src="https://user-images.githubusercontent.com/46443482/193080443-26277897-4e4d-4699-af6e-e9e6a211c06d.png">
@vmartins

**Reply by vmartins** [reply]
After commit a833a88 (Laravel >= v9.27.0), I think you need to translate this:

- ```(and :count more error)```
- ```(and :count more errors)```

**Reply by Pablo-Camara** [reply]
My problem is that I was doing it in the validation.php, had to do it in the lang/pt.json ^^

**Comment by skarjoss** [comment]
Before v9.27.0, my solution was to override the error message in the exception handler.

`
// In file app/Exceptions/Handler.php add the render method

    public function render($request, Throwable $e)
    {
        // Grab the HTTP status code from the Exception, or default response of 400
        $statusCode = $this->isHttpException($e) ? $e->getStatusCode() : 400;

        // If is validation exception and json request
        if ($e instanceof ValidationException && $request->wantsJson())
        {
            // Apply pluralization
            $messages = $e->validator->errors()->all();
            $message = array_shift($messages);
            if ($additional = count($messages)) {
                $pluralized = $additional === 1 ? __('error') : __('errors');
                $message .= __(" (and :additional more :pluralized)", compact('additional', 'pluralized'));
            }
            // Send pluralized message with errors
            return response()->json(['message' => $message, 'errors' => $e->errors()], $statusCode);
        }

        // Default to the parent handler class implementation
        return parent::render($request, $e);
    }



`
